### PR TITLE
add jquery-price-format w/ git autoupdate

### LINKED ISSUE #8048: [Request] Add Jquery-Price-Format
**Library name:** Jquery-Price-Format
**Git repository url:** https://github.com/flaviosilveira/Jquery-Price-Format
**npm package url(optional):** 
**License(s):**
**Official homepage:** http://jquerypriceformat.com/
**Wanna say something? Leave message here:**

=====================
Notes from cdnjs maintainer:
Please read the README.md and CONTRIBUTING.md document first.

You are welcome to add a library via sending pull request,
it'll be faster then just opening a request issue,
and please don't forget to read the guidelines for contributing, thanks!!

### PR DESCRIPTION
**Pull request by pvnr0082t**
PR for #8048 
@kennynaoh Please help me review this pull request, thank you.
Manually add jquery-price-format v2.0 because naming scheme is different from the latest version.

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib 
 * Git repository (required):https://github.com/flaviosilveira/Jquery-Price-Format
 * Official website (optional, not the repository): http://www.jquerypriceformat.com/
 * NPM package url (optional):
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 167
   - Count of watchers: 9
   - Count of forks: 89
 * NPM download stats (optional):
   - Downloads in the last day:
   - Downloads in the last week:
   - Downloads in the last month:

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [ ] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8048 , cc @flaviosilveira

### CODE REVIEW

**Comment by kennynaoh** [comment]
@pvnr0082t 
I think `v2.0` is same with `v2.0.0`.
I found that `jquery.price_format.js` in these two versions are the same.
And the version name is confused.
Is it possible just add one of them?
cc @PeterDaveHello

**Comment by PeterDaveHello** [comment]
Maybe you can discuss with the author to make sure if they are totally the same, if so, dropped one of them, and add the path to [.gitignore](https://github.com/cdnjs/cdnjs/blob/master/.gitignore), thanks.

**Comment by pvnr0082t** [comment]
@kennynaoh I have removed v2.0.0 and add it in `.gitignore`. Please help me review it again, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Multi commits PR @cdnjs/intern2

**Comment by x09326** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
@x09326 There is conflicts, so not good at all.

**Comment by pvnr0082t** [comment]
@kennynaoh @x09326 I fixed the conflict. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by x09326** [comment]
LGTM

File: .gitignore [hunk 1]
```diff
@@ -14,6 +14,7 @@ node_modules/
 ajax/libs/*/*/.bot_cant_auto_add
 
 # folders need to be ingored due to auto auto-update mechanism
+ajax/libs/jquery-price-format/2.0.0/
```
Comment (PeterDaveHello) [inline .gitignore]: What is this for? There is no `2.0.0` tag in repo and you added v2.0, so no need to ignore it,

**Comment by pvnr0082t** [comment]
I didn't notice that tag v2.0.0 removed. I have fixed it. @kennynaoh @x09326 Please help me review again, thank you.

**Comment by kennynaoh** [comment]
I think you don't need to mention `ignore v2.0.0` in commit message now.
Thanks!

**Comment by pvnr0082t** [comment]
@kennynaoh @x09326 I have fixed commit message. Please help me review again, thank you.

**Comment by kennynaoh** [comment]
LGTM

**Comment by x09326** [comment]
LGTM

**Comment by PeterDaveHello** [comment]
Thanks!

**Pull request merged**